pkg/didcomm/crypto/jwe/authcrypt/decrypt.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package authcrypt
+
+import (
+	"golang.org/x/crypto/blake2b"
+	"golang.org/x/crypto/chacha20poly1305"
+	"golang.org/x/crypto/nacl/box"
+)
+
+// Decrypt will JWE decode the envelop argument for the sender and recipients
+// Using (X)Chacha20 encryption algorithm and Poly1035 authenticator
+func (c *Crypter) Decrypt(envelope []byte) (string, error) {
+	decryptOID(nil, nil, nil)
```

Comment:
I assume this is a TODO? If so, please add code comments and create issue.

Reply:
yes will add a TODO

Reply:
done